examples/mnit_ballz/src/objects.nit

```diff
@@ -0,0 +1,90 @@
+# Module containing all objects of the game
+module objects
+
+import geometry
+import geometry::polygon
+import geometry::boxes
+import collision
+
+# The ball is the main character of the game !
+class Ball
+	#center of the ball
+	var center: Point[Float] is writable
+end
+
+# Walls make the ball bounce on them
+class Wall
+	# coordinates of the center of the wall
+	var center: Point[Float]
+
+	# angle in degrees
+	var angle: Float
+
+	# Scale for drawing `self`
+	var scale: Float is noautoinit
+
+	# Width of `self`
+	var width: Float is noautoinit
+
+	# Height of `self`
+	var height: Float is noautoinit
+
+	# Lines composing `self`
+	var lines: Array[OrientedLine[Float]] is noautoinit
+
+	# Initialize `self` with all its lines from `center` and `angle`
+	init complete(center: Point[Float], angle: Float,  scale: Float) do
```

Comment:
why a named constructor here? Remove `noautoinit` in center, angle and scale and win a free autoinit constructor